katas/content/superposition/three_states_two_qubits_phases/Solution.qs

```diff
@@ -0,0 +1,23 @@
+namespace Kata {
+    open Microsoft.Quantum.Math;
+
+    operation ThreeStates_TwoQubits_Phases (qs : Qubit[]) : Unit {
+        // First create (|00⟩ + |01⟩ + |10⟩) / sqrt(3) state
+        ThreeStates_TwoQubits_Reference(qs);
+
+        R1(4.0 * PI() / 3.0, qs[0]);
+        R1(2.0 * PI() / 3.0, qs[1]);
+    }
+
+    operation ThreeStates_TwoQubits_Reference (qs : Qubit[]) : Unit is Adj {
```

Comment:
This will look better if the solution uses the same operation name as the placeholder - this way the learner can copy-paste just the block of three commands from ThreeStates_TwoQubits_Phases to their solution, and not edit them before running
```suggestion
    operation ThreeStates_TwoQubits (qs : Qubit[]) : Unit is Adj {
```